_episodes/05-creating-container-images.md

```diff
@@ -117,8 +117,22 @@ Let's break this file down:
 - The first line, `FROM`, indicates which container we're starting with.
 - The next two lines `RUN`, will indicate installation commands we want to run. These
 are the same commands that we used interactively above.
-- The last line, `CMD` indicates the default command we want the container to run,
-if no other command is provided.
+- The last line, `CMD`, indicates the default command we want the
+container to run, if no other command is provided. It is recommended
+to provide `CMD` in *exec-form*, that is a JSON-array which contains
```

Comment:
I'd prefer not to introduce more terms that people won't know (JSON-array) in this case, but I don't know if there's a good alternative phrasing. If we use the term JSON-array, I think we need to explain something about what that means or what it looks like, even something as literal as "a list of text strings in square brackets".